demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -234,20 +238,44 @@ def verify_no_empty_sections(self) -> bool:
 
         return is_valid
 
+    def _find_section_in_text(self, sections_list: List[str], ignore_packs: Optional[List[str]] = None) -> str:
+        """
+        Find if sections from the sections list appear in the readme content and returns an error message.
+        Arguments:
+            sections_list - list of strings, each string is a section to find in the text
+            ignore_packs - List of packs and integration names to be ignored
+        Returns:
+            An error message with the relevant sections.
+        """
+        errors = ""
+
+        current_pack_name = self.pack_path.name
+        if ignore_packs and current_pack_name in ignore_packs:
+            print("\n\n ignoring this pack \n\n")
```

Comment:
please use `click.secho` and print this in yellow as this is a warning.

Reply:
Done